### ISSUE TITLE
Choosing an alternative to Parse

### ISSUE THREAD

**Issue by ky1ejs** [kylejm/Jim#3]
I think I'm liking the look of either [Couchbase](http://developer.couchbase.com/mobile) or [CloudKit](https://developer.apple.com/icloud). 

Couchbase would be particularly nice as it's open source and I hear it handles offline access and sync very nicely.

Brilliant list of alternatives [here](https://github.com/relatedcode/ParseAlternatives/#open-source-solutions) by @relatedcode.

**Comment by idekov-hf** [comment]
Hey @kylejm have you managed to choose an alternative to Parse for Jim yet?

**Comment by ky1ejs** [comment]
Hey @ivdekov, thanks for getting in touch.

Not yet, no. I'm currently quite busy at work and am lucky enough to be going to WWDC next week. Hopefully I can look at some stuff while travelling though.

If you have any suggestions, I'd love to hear them :)

**Comment by idekov-hf** [comment]
Have you considered deploying your own parse server rather than finding an alternative?

**Comment by ky1ejs** [comment]
@ivdekov sorry for the delay in my reply. Busy getting back to work after WWDC 😅.

I'm not too sure about the future of Jim. I have very little time for it ( 😢 ), and there is a very serious competitor in [Strong](https://itunes.apple.com/us/app/strong-workout-tracker/id464254577?mt=8).

I've met with Cameron (creator of Strong) a few months ago, he's an awesome guy. He makes a living off Strong, so open sourcing it is a bit more of a complicated. I would love him to do so, and then I'd divert attention on Jim to Strong.

Back to your question, I think I'd rather give CloudKit a go, but looking in to the cons and pros more would be excellent. Parse Server would also be an option of course. If you have any particular reasons for recommendation, that would be cool :).

**Comment by idekov-hf** [comment]
Not a problem for the delay @kylejm.

Is most of your time going towards your work?

I was just wondering if you had considered Parse Server is all. I'm still trying to learn about it and integrate it into my own app. Having it (and a Parse Dashboard) deployed on Heroku is very convenient and the only downside that I have experienced so far is having to wait for a bit when accessing the server and dashboard after 30 minutes of inactivity while Heroku starts up those apps. 

Besides that, I like that you can send Push Notifications through API calls to the Parse Server. That's very handy for what I have been working on.

**Comment by ky1ejs** [comment]
Yeah, a lot of my time is being spent on work right now - we have a major release coming up in a couple of months.

I think Parse Server is definitely a strong contender, but I'm very aware of cost though, Heroku can get pricy. Keeping things free is definitely a priority for this OSS project.
